src/custom/pages/Swap/SwapMod.tsx

```diff
@@ -878,19 +873,15 @@ export default function Swap({
                   </Text> */}
               </ButtonError>
             )}
-            {/* {showApproveFlow && (
-              <Column style={{ marginTop: '1rem' }}>
-                <ProgressSteps steps={[approvalState === ApprovalState.APPROVED]} />
-              </Column>
-            )} */}
             {isExpertMode && swapErrorMessage ? <SwapCallbackError error={swapErrorMessage} /> : null}
           </BottomGrouping>
         </Wrapper>
       </AppBody>
       {/* <SwitchLocaleLink /> */}
       {!swapIsUnsupported ? null : !isSupportedWallet ? (
         <UnsupportedCurrencyFooter
-          show={!isSupportedWallet}
+          show={swapIsUnsupported}
```

Comment:
Guess you missed the `!isSupportedWallet` flag here

Reply:
@alfetopito don't get this comment, did i make a mistake or